### ISSUE TITLE
Does PINN training require supervised learning of reference solutions to training data points?

### ISSUE THREAD

**Issue by lmtmtmt** [lululxvi/deepxde#1729]
Hello, Doctor, in the case of the "Helmholtz acoustic hard scattering problem with absorption boundary conditions", the analytical solution was not given and the training result was wrong, why? Is it possible to train in PINN only through the boundary conditions and the loss function of the PDE term, and will the training result be greatly affected by not giving the analytic solution?

**Comment by praksharma** [comment]
PINNs were originally proposed to replace PDE solvers, meaning no analytical solution is required.  This is very different from conventional supervised PDE solvers such as unet, FEDformer, FNO where you actually provide the analytical solution for each sample.
Theoretically, a PINN should work just like a PDE solver. But in practice, there are so many problems that an accurate solution is not guaranteed. You need to tinker the hyperparameters.
